x-pack/plugins/index_management/public/application/components/component_templates/component_template_selector/components/filter_list_button.tsx

```diff
@@ -76,7 +83,10 @@ export function FilterListButton({ onChange, filters }: Props) {
         panelPaddingSize="none"
         data-test-subj="filterList"
       >
-        <div className="euiFilterSelect__items">
+        {/* EUI NOTE: Please use EuiSelectable (which already has height/scrolling built in)
```

Comment:
Is there an approximate timeline yet for the deprecation of `EuiFilterSelectItem`?

Reply:
Apologies for not clarifying this - we're shooting for the end of the 2023, although this isn't a firm timeline on our end and we're super happy to flex/work with Kibana teams on this!